tests/containers/podman_integration.pm

```diff
@@ -12,19 +12,46 @@ use testapi;
 use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
 use containers::utils qw(get_podman_version);
 use utils qw(zypper_call script_retry);
-use version_utils qw(get_os_release);
+use version_utils qw(get_os_release is_transactional is_sle is_sle_micro is_tumbleweed);
+use transactional qw(trup_call check_reboot_changes);
+use registration qw(add_suseconnect_product get_addon_fullname);
 use containers::common;
 
 sub run {
     select_serial_terminal;
     my ($running_version, $sp, $host_distri) = get_os_release;
     install_podman_when_needed($host_distri);
 
+    if (is_sle_micro) {
+        my $sle_version = "";
+        if (is_sle_micro('<5.3')) {
+            $sle_version = "15.3";
+        } elsif (is_sle_micro('<5.5')) {
+            $sle_version = "15.4";
+        } elsif (is_sle_micro('<6.0')) {
+            $sle_version = "15.5";
+        }
+        trup_call "register -p PackageHub/$sle_version/" . get_required_var('ARCH');
+        check_reboot_changes;
```

Comment:
If you use trup with `-c` later, you only need a single reboot

Reply:
Fixed.